libraries/lib-files/PathList.cpp

```diff
@@ -20,6 +20,9 @@ void FileNames::InitializePathList()
    auto &standardPaths = wxStandardPaths::Get();
    const auto programPath = standardPaths.GetExecutablePath();
 
+   // encouraged by wxwidgets
+   standardPaths.SetFileLayout(wxStandardPaths::FileLayout::FileLayout_XDG);
```

Comment:
Probably it should go to `#if defined(__WXGTK__)` as well

Reply:
Will do.